spec/teacher_spec.rb

```diff
@@ -0,0 +1,9 @@
+require_relative '../teacher'
+describe Teacher do
+  context 'When testing the Teacher class' do
+    it 'Should always return true on can_use_services? method' do
+      teacher = Teacher.new(22, 'CS', 'John')
+      expect(teacher.can_use_services?).to be true
+    end
+  end
```

Comment:
- [ ] Kindly add test to check when initializing the `Teacher` class (test for constructor method) as you rightly did for `book_spec.rb` and `person_spec.rb` files. 👍